### DISCUSSION TITLE
Why our videojs plugin is not included in videojs.com/plugins ?

### DISCUSSION THREAD

**Discussion by thomasjammet** [videojs/video.js#8610] [General]
Hello community!

We have built this new videojs plugin and would like it to be available in the official videojs page :
https://www.npmjs.com/package/@ceeblue/videojs-plugins

Why isn't it included in videojs.com/plugins? It is public and has the `videojs-plugin` keyword.

Thank you

**Comment by mister-ben** [comment]
It's tagged correctly. It will show up when the site is next deployed. Thanks for contributing.

**Reply by thomasjammet** [reply]
Great! Thank you for the fast answer!

**Reply by mister-ben** [reply]
It's there now

**Comment by thomasjammet** [comment]
Awesome, thank you Mister Ben!
